rosetta/retriever/invoker.go

```diff
@@ -16,8 +16,10 @@ package retriever
 
 import (
 	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go/model/flow"
 )
 
 type Invoker interface {
 	Script(height uint64, script []byte, parameters []cadence.Value) (cadence.Value, error)
+	GetAccount(address flow.Address, height uint64) (*flow.Account, error)
```

Comment:
Nooooo. Please re-read https://golang.org/doc/effective_go#Getters.

What's the advantage of `invoke.GetAccount` over `invoke.Account`?

Reply:
I was just following suite with the existing code that was `GetAccount` 😃 
I've renamed `GetAccount` to `Account` in other places too.